.env

```diff
@@ -3,5 +3,10 @@ MONGO_HOST=localhost
 MONGO_PORT=27017
 MONGO_DATABASE=mongoose
 
+MG_API_KEY=989a6f8aba94eade321e66c815d2b73d-29561299-2ff04a3b
```

Comment:
Uważajcie na to. Bo teraz skoro to działający API_KEY a repo jest publiczne, to każdy może wysłać maila z waszego konta :)